resolwe_bio/kb/serializers.py

```diff
@@ -19,3 +19,4 @@ class Meta:
         """Serializer configuration."""
 
         model = Feature
+        fields = '__all__'
```

Comment:
Why not? Search endpoints are all read-only and the REST endpoints are admin-only.